addon/-private/system/model/errors.js

```diff
@@ -96,6 +96,34 @@ var MapWithDefault = Ember.MapWithDefault;
   {{/each}}
   ```
 
+  The JSON API spec also allows for object level errors to be placed
+  in an object with pointer `data`
+
+  ```javascript
+  {
+    "errors": [
+      ...
+	  {
+        "detail": "Some generic non property error message",
+	      "source": {
+		    "pointer": "data"
+	      }
+	  }
+	]
+  }
+  ```
```

Comment:
I am not sure if this is necessary or if it shows up correctly nevertheless in the API docs, but can you reformat this so the indentation is correct for this code sample? Also, I think the `...` might interfere with the API docs since it's not a valid JS syntax, so maybe they should be removed? What do you think?